api/helpers/gamestate/moves/points/execute.js

```diff
@@ -0,0 +1,45 @@
+// api/helpers/gameState/moves/points/execute.js
+const _ = require('lodash');
+
+module.exports = {
+  friendlyName: 'Play Points',
+
+  description: 'Returns new GameState resulting from requested points move',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest game state before the requesting player plays a card for points',
+      required: true,
+    },
+    /**
+     * @param {Object} requestedMove - Object describing the request to play points
+     * @param {1 | 0} requestedMove.playedBy - Which player is playing
+     * @param { Card } requestedMove.cardPlayed - Card Played for points
+     * @param { MoveType.POINTS } requestedMove.moveType - Specifies that this a Points move
+     */
+    requestedMove: {
+      type: 'ref',
+      description: 'The move being requested. Specifies which player is asking to play a card for points',
+    },
+  },
+  sync: true, // synchronous helper
+  fn: ({ currentState, requestedMove }, exits) => {
+    const { playedBy, cardPlayed } = requestedMove;
+    let result = _.cloneDeep(currentState);
```

Comment:
We could alternatively use `structuredClone` here instead of having to import lodash

Reply:
Just make sure it exists in the node version we're using.

Reply:
Oohhh, good call I didn't even think to check that.

We're running 18, and structuredClone was added in 17